config.json.example

```diff
@@ -0,0 +1,16 @@
+{
+    "urls": {
+        "Rest service": {
+            "url": "https://example.com/app/to/check",
```

Comment:
If they're not set at all the bot always runs because it checks for `undefined`. It only checks for the date bounds if you pass them. Passing them in the wrong format is something else, this is a very simple and straightforward script, I'm not sure how much it's worth to check for the correct format of the dates. I can add a check to see if the time passed is between 0 and 2400, but that's pretty much it

Reply:
I added this to the `README.md` for this very reason - `The date fields and mention field
are all optional`

Reply:
true